packages/clerk-auth/src/index.ts

```diff
@@ -1,14 +1,13 @@
-import type { ClerkOptions } from '@clerk/backend'
-import { Clerk, createIsomorphicRequest, constants } from '@clerk/backend'
+import { type ClerkClient, type ClerkOptions, createClerkClient } from '@clerk/backend'
+import type { RequestState } from '@clerk/backend/dist/tokens/authStatus'
```

Comment:
```suggestion

```
It's better not to import the type from dist but infer it instead, I have left another comment bellow